onnxruntime/core/providers/webnn/builders/impl/normalization_op_builder.cc

```diff
@@ -86,12 +110,42 @@ Status NormalizationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder
   NodeAttrHelper helper(node);
   options.set("epsilon", helper.Get("epsilon", 1e-05f));
 
-  int64_t axis = helper.Get("axis", -1);
-  axis = HandleNegativeAxis(axis, rank);
-  std::vector<int32_t> axes{static_cast<int32_t>(axis)};
-  options.set("axes", emscripten::val::array(axes));
-
-  emscripten::val output = model_builder.GetBuilder().call<emscripten::val>("meanVarianceNormalization", input, options);
+  emscripten::val output = emscripten::val::undefined();
+  if (op_type == "LayerNormalization") {
+    int64_t axis = helper.Get("axis", -1);
+    axis = HandleNegativeAxis(axis, rank);
+    std::vector<int32_t> axes{static_cast<int32_t>(axis)};
```

Comment:
This only sets one axis in axes, but the [ONNX docs say LayerNormalization's `axis`](https://github.com/onnx/onnx/blob/main/docs/Operators.md#attributes-47) is the first axis. So a 4D input tensor with axis = 1 would need to pass {1,2,3}.

```
std::vector<int32_t> axes(rank - axis);
std::iota(axes.begin(), axes.end(), axis);
```